examples/ping/src/lib.rs

```diff
@@ -28,3 +28,34 @@ pub use code::WASM_BINARY_OPT as WASM_BINARY;
 
 #[cfg(not(feature = "std"))]
 mod wasm;
+
+#[cfg(test)]
```

Comment:
do we really need this test module? :/

Reply:
you mean no tests for `demo_ping` or without this module but embed `gtest` imports in the tests?

Reply:
I see no reason in introducing these tests

Reply:
https://github.com/gear-tech/gear/pull/3419/commits/d6529ba5205a6f1a6a7eae9377a77f81816f01b4 removed